apps/show/helpers/view_helpers.coffee

```diff
@@ -239,6 +240,12 @@ module.exports =
       ]).join(' ')
     ]).join(', ')
 
+  fairOrShowLocation: (show) ->
+    if show.fair?
+      show.fair.location
+    else
+      show.location
```

Comment:
When the show is a partner show, we want it's location. When it's a fair booth, we want the the location of the fair.